pages/en/glossary.mdx

```diff
@@ -0,0 +1,81 @@
+---
+title: Glossary
+---
+
+- **The Graph**: A decentralized protocol for indexing and querying data from blockchains.
+
+- **Query**: A request for data from a database. In the case of The Graph, a query is a request for data from a subgraph that will be answered by an Indexer.
```

Comment:
I'd skip the database part. That's a general explanation but a query can be much more than just a request to a database.

Reply:
Didn't know that, thank you!